### ISSUE TITLE
Support css attribute parsing

### ISSUE THREAD

**Issue by wizztjh** [reworkcss/css#78]
Feature request for adding css attribute parsing.

example

`style="border-width: 1px"

**Comment by faywong** [comment]
+1
I also need this feature in my css parsing.

**Comment by leebenson** [comment]
In the interim, I'm bolting this on to 'fake' selectors like ``css.parse(`*{${style}}`);``, and removing the `*{}` from the stringified result. Not optimal, but it works.

**Comment by AkatQuas** [comment]
> In the interim, I'm bolting this on to 'fake' selectors like `` css.parse(`*{${style}}`); ``, and removing the `*{}` from the stringified result. Not optimal, but it works.

Indeed, that's how the `vscode-html-language` works with inline style and `<style></style>`  validation. 

But it requires to parsing html tokens.
